src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/NuGetReference.cs

```diff
@@ -7,6 +7,8 @@ namespace Microsoft.DotNet.UpgradeAssistant
 {
     public record NuGetReference(string Name, string Version)
     {
+        public bool IsPrerelease => Version.Contains("-");
```

Comment:
Can you add a comment as to why this is a good check for IsPrerelease? I'm assuming it's per semantic versioning, but it would be good to document the why.

Reply:
Yes, semver (https://semver.org/spec/v2.0.0.html#spec-item-9) and a comment here is a good idea. I'll add one.